exercises/concept/flight-to-mars/flight_to_mars.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class LaunchStatusCheck
+  # create a custom `LaunchStatusError` error
+
+  # create a custom `AbortLaunchError` error
+
+  def initialize(flight_teams)
+    raise NotImplementedError, 'Please implement the LaunchStatusCheck#initialize method'
```

Comment:
In other discussion, we have decided to not use the `NotImplementedError` since it implies and is intended for those things not implemented "on this platform" rather than a simple "not available to use".  I would recommend a `NoMethodError` with the message to cause the desired method to exist.